tools/test/h5dump/testh5dump.sh.in

```diff
@@ -868,6 +870,35 @@ TOOLTEST5() {
     fi
 }
 
+# same as TOOLTEST1 but expects h5dump to fail
+#
+TOOLTEST_FAIL() {
+
+    infile=$1
+    expect="$TESTDIR/`basename $1 exp`.ddl"
+    actual="$TESTDIR/`basename $1 .exp`.out"
+
+    # Run test.
+    TESTING $DUMPER $@
+    (
+        cd $TESTDIR
+        $RUNSERIAL $DUMPER_BIN "$@" $infile
+    ) >&$actual
+    RET=$?
+    # Segfault occurred
+    if [ $RET == 139 ] ; then
+        nerrors="`expr $nerrors + 1`"
+        echo " FAILED"
```

Comment:
I think it might be worthwhile to differentiate the two "FAILED" messages to make the log more meaningful and also to make clear in the second case that an expected h5dump failure didn't happen.  Also note that all other FAILED messages in the script are surrounded by *s:  *FAILED*.  If you follow that convention it makes it easier to distinguish failed tests in the logs from incidental occurrences of "FAILED" that don't indicate test failures. 

I suggest:
line 891 "*FAILED* - [h5dump] test failed with segmentation fault"
line 895 "*FAILED* - [h5dump] test did not fail as expected"

[ ] - optional - could also be test name if available.

Reply:
I've made the changes and resolved the conflict.